silta-cicd/circleci-php8.2-node18-composer2/README.md

```diff
@@ -0,0 +1,14 @@
+# silta-circleci
+A docker image used circleCI, based on `cimg/php:8.1.7-node` with the following additions:
+
+- Composer configured correctly
+- Drush-launcher and coder pre-installed
+- Vim, useful for debugging
+- The google cloud cli, kubernetes and helm
+
+## Versions
+- PHP: 8.1.7
```

Comment:
I guess this should be 8.2.0 or something?

Reply:
this was branched from #81, it will autoresolve or get fixed when the other is merged